Content.Shared/DoAfter/SharedDoAfterSystem.cs

```diff
@@ -65,7 +65,8 @@ private void OnDamage(EntityUid uid, DoAfterComponent component, DamageChangedEv
     {
         // If we're applying state then let the server state handle the do_after prediction.
         // This is to avoid scenarios where a do_after is erroneously cancelled on the final tick.
-        if (!args.InterruptsDoAfters || !args.DamageIncreased || args.DamageDelta == null || GameTiming.ApplyingState)
+        if (!args.InterruptsDoAfters || !args.DamageIncreased || args.DamageDelta == null || GameTiming.ApplyingState
+            || args.DamageDelta.DamageDict.ContainsKey("Radiation")) // imp - Sanity check so people can crowbar doors open to flee from Lord Singuloth
```

Comment:
this isnt needed it was fixed like 2 years ago, radiation doesnt interrupt doafters